.changeset/five-mayflies-compare.md

```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/react-components": patch
+"@wso2is/theme": patch
```

Comment:
I think we need to bump, console and myaccount versions as well ? Right @JayaShakthi97

Reply:
Fixed with 2fb7b58